recipes/opencl-headers/all/conanfile.py

```diff
@@ -31,7 +31,15 @@ def package(self):
     def package_info(self):
         self.cpp_info.filenames["cmake_find_package"] = "OpenCLHeaders"
         self.cpp_info.filenames["cmake_find_package_multi"] = "OpenCLHeaders"
+        self.cpp_info.set_property("cmake_file_name", "OpenCLHeaders")
+
         self.cpp_info.names["cmake_find_package"] = "OpenCL"
         self.cpp_info.names["cmake_find_package_multi"] = "OpenCL"
+        self.cpp_info.set_property("cmake_target_name", "OpenCL::OpenCL")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_target_name", "OpenCL::OpenCL")
```

No

Reply:
can you explain that? from the [docs](https://github.com/conan-io/conan-center-index/blob/master/docs/preparing_recipes_for_conan2.md#translating-names-information-to-cmake_target_name-cmake_module_target_name-and-cmake_file_name) it seems that this recipe generates the target `OpenCL::OpenCL` for both the `cmake_find_package` and the `cmake_find_package_multi` generators.

Reply:
It's an unwanted side effect of cmake_find_package, the truth is OpenCL source code.